leveldbjni/src/main/java/org/fusesource/leveldbjni/internal/JniDB.java

```diff
@@ -219,59 +219,15 @@ public void suspendCompactions() throws InterruptedException {
         if( db==null ) {
             throw new DBException("Closed");
         }
-        db.suspendCompactions();
+	// Not implemented
```

Comment:
Can you add your github id in parentheses and describe why you comment out the code here?

```
// (github id) To fix issue https://github.com/utsaslab/pebblesdb/issues/2, ... (briefly describe your reasoning here)
```

Reply:
Sure, will do this.